spring-cloud-aws-s3-parent/spring-cloud-aws-s3-config/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need this in separate module? Perhaps it can be added to S3 module?

Reply:
@maciejwalkowiak @MatejNedic 

I am not sure but for S3 only we have made separate module for config and there is only one class there that is `S2PropertySource`.  Not sure intention behind adding new module. If not need we can move both pom and above class to S3 module under S3-Parent.